backend/src/main/java/ca/bc/gov/hlth/mohums/webclient/WebClientConfig.java

```diff
@@ -26,18 +25,18 @@
 
     @Bean
     public ReactiveOAuth2AuthorizedClientManager authorizedClientManager(
-            final ReactiveClientRegistrationRepository clientRegistrationRepository,
-            final ServerOAuth2AuthorizedClientRepository authorizedClientRepository) {
+            ReactiveClientRegistrationRepository clientRegistrationRepository,
+            ReactiveOAuth2AuthorizedClientService authorizedClientService) {
 
-        ReactiveOAuth2AuthorizedClientProvider authorizedClientProvider
-                = ReactiveOAuth2AuthorizedClientProviderBuilder.builder()
+        var authorizedClientProvider =
+                ReactiveOAuth2AuthorizedClientProviderBuilder.builder()
                         .clientCredentials()
                         .build();
 
-        DefaultReactiveOAuth2AuthorizedClientManager authorizedClientManager
-                = new DefaultReactiveOAuth2AuthorizedClientManager(clientRegistrationRepository,
-                        authorizedClientRepository);
-
+        var authorizedClientManager =
+                new AuthorizedClientServiceReactiveOAuth2AuthorizedClientManager(
+                        clientRegistrationRepository,
+                        authorizedClientService);
```

Comment:
This change was necessary to prevent a runtime exception when calling `block` on an exchange:

Exception "serverWebExchange cannot be null".

Reference:
* https://github.com/spring-projects/spring-security/issues/8230
* https://github.com/spring-projects/spring-security/issues/7468
* https://gist.github.com/abhi2495/51ad47394da2effc0fb63f97b3a78e37#gistcomment-3390204